### PR TITLE
[POU-87] Responsive banner image

### PR DESCRIPTION
**Pull request by MingLiang-PALOIT**
Task: [AB#POU-87](https://cadtrust.atlassian.net/browse/POU-87)

# Why

To create a responsive header that adapts to the desktop, tablet and mobile

# What

The reactive `AppHeader` is placed under `organisms` with an accompanying `AppHeader.module.scss`
This has been rebased off main.
A snapshot test has been written

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -3,4 +3,5 @@ module.exports = {
   trailingComma: 'all',
   semi: false,
   printWidth: 180,
-}
+  tabWidth: 2,
```
Comment (morphet81) [inline .prettierrc.js:6]: Why did you add this? It's the default value, shouldn't be necesary

File: src/assets/header.jpg
Comment (morphet81) [inline src/assets/header.jpg:1]: Optimize the file weight

File: src/components/organisms/AppHeader/AppHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  })
+
+  return (
+    <Box>
+      <Flex backgroundImage={headerBanner.src} aspectRatio={[`1.5`, `2`, `3`, `4`, `5`]} className={styles.backgroundImage}>
+        <Box width={[`100vw`, `60vw`]} className={styles.backgroundGradient} />
+        <Flex className={styles.headerContents}>
+          <Flex direction={`column`} gap={`20px`} alignSelf={[`top`, null, `center`]} padding={[`50px 20px`, null, `5%`]}>
+            <Text fontSize={[`lg`, `xl`, `3xl`]} className={styles.headerTitle}>
+              Connecting Carbon markets through open data.
```
Comment (morphet81) [inline src/components/organisms/AppHeader/AppHeader.tsx:24]: Localization

File: src/components/organisms/AppHeader/AppHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            </Text>
+            <Link as={`button`} variant={`whiteSecondary`} href={`https://www.google.com`}>
+              Find out more about CAD Trust
```
Comment (morphet81) [inline src/components/organisms/AppHeader/AppHeader.tsx:27]: Localization

File: src/components/organisms/AppHeader/AppHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+            </Link>
+          </Flex>
+          <Spacer />
+          <Flex>
+            <Image {...headerFeatureImage} height={`100%`} alt={`header feature image`} />
```
Comment (morphet81) [inline src/components/organisms/AppHeader/AppHeader.tsx:32]: Alts should be localized as well

**Review by morphet81 — CHANGES_REQUESTED** [review]

File: .prettierrc.js (hunk 1 above)
Comment (MingLiang-PALOIT) [reply .prettierrc.js:6]: had issues with the ide not picking up previously. works now, will remove.

**Review by morphet81 — CHANGES_REQUESTED** [review]
I was hoping you would review all `alt` props. You missed some.

**Review by morphet81 — APPROVED** [review]
